website/interview_exp/api/views.py

```diff
@@ -181,7 +183,7 @@ def perform_update(self, serializer):
 class RevisionsListView(ListAPIView):
     serializer_class = RevisionSerializer
 
-    permission_classes = (IsMemberOrAbove,)
+    # permission_classes = (IsMemberOrAbove,)
```

Comment:
why is permission_classes commented out ?

Reply:
This one was done by mistake fixed it now.